systers_portal/templates/meetup/list_location.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block title %}
+  Systers Meetup Locations
+{% endblock %}
+
+{% block content %}
+  <h2 class="mt40 text-center">Systers Meetup Locations</h2>
+  <div class="row">
+    <div class="col-xs-offset-1 col-xs-10 col-xs-offset-1 mt20">
```

Comment:
Just a suggestion: what if we leave the map width to take the whole available space and increase the height a bit, say 400px.